src/cells/memory.mjs

```diff
@@ -10,84 +10,110 @@ import { Vector3vl, Mem3vl } from '3vl';
 
 // Memory cell
 export const Memory = Box.define('Memory', {
+    /* default properties */
+    bits: 1,
+    abits: 1,
+    rdports: [{clock_polarity: true}],
+    wrports: [{}],
```

Comment:
Wrong default: the default should be no read/write ports at all. The reason is that if the JSON specifies no ports, there should be no ports.

Reply:
True, I guess I've forgotten to remove these defaults after testing. Good catch!